scripts/publish-version.sh

```diff
@@ -63,6 +63,10 @@ declare environment_id
 for git_ref in $(cat "${mydir}/../packages/hoprd/releases.json" | jq -r "to_entries[] | .value.git_ref" | uniq); do
   if [[ "${branch}" =~ ${git_ref} ]]; then
     environment_id=$(cat "${mydir}/../packages/hoprd/releases.json" | jq -r "to_entries[] | select(.value.git_ref==\"${git_ref}\" and .value.default==true) | .key")
+    # if no default is set we take the first entry
```

Comment:
Why?

Reply:
So we don't have to explicitly set defaults in the configuration for one entry only.